src/material/slider/slider-input.ts

```diff
@@ -215,6 +215,11 @@ export class MatSliderThumb implements _MatSliderThumb, OnDestroy, ControlValueA
 
   /** Used to relay updates to _isFocused to the slider visual thumbs. */
   private _setIsFocused(v: boolean): void {
+    // The slider never receives focus in safari.
+    if (this._platform.SAFARI) {
```

Comment:
I'm not sure that we can do this. My understanding is that users have to enable focus in Safari.

Reply:
How do you enable this? I can't find a way to enable focus on `input type="range"` on safari

Reply:
So there's a setting at `Safari > Settings > Advanced > Press tab to highlight each item on the page` which I believe toggles whether things are focusable.

Reply:
I pushed a new approach which avoids breaking this edge case. One thing I noticed is that even with this setting enabled, the slider still does not maintain focus from pointer events